stakedb/ticketpool.go

```diff
@@ -137,11 +137,11 @@ func (l *badgerLogger) Errorf(format string, v ...interface{}) {
 
 var versionKey = []byte("version")
 
-const currentVersion uint32 = 1
+const latestVersion uint32 = 3
```

Comment:
This doesn't refer to the badger version, but our scheme.  Should just go to 2.

Reply:
Ah, okay.